rtengine/imageio.cc

```diff
@@ -964,14 +1003,26 @@ int ImageIO::savePNG  (Glib::ustring fname, int compression, volatile int bps)
     png_set_IHDR(png, info, width, height, bps, PNG_COLOR_TYPE_RGB,
                  PNG_INTERLACE_NONE, PNG_COMPRESSION_TYPE_DEFAULT, PNG_FILTER_TYPE_BASE);
 
-
     int rowlen = width * 3 * bps / 8;
     unsigned char *row = new unsigned char [rowlen];
 
+    const bool do_pseudogrey = bps == 8 && isBW();
```

Comment:
It has of course nothing to do with this change, but why the heck is `bps` volatile?